src/pages/registration/resource_type_tab/sub_type_forms/artistic_types/OutputRow.tsx

```diff
@@ -107,25 +107,19 @@ export const OutputRow = ({
         <Typography>{title}</Typography>
       </TableCell>
       <TableCell>
-        {maxIndex !== 0 && (
-          <Box
-            sx={{ display: 'grid', gridTemplateAreas: '"down up"', gridTemplateColumns: '1fr 1fr', maxWidth: '8rem' }}>
-            {index !== maxIndex && (
-              <Tooltip title={t('common.move_down')} sx={{ gridArea: 'down' }}>
-                <Button onClick={() => moveItem(index + 1)}>
-                  <ArrowDownwardIcon />
-                </Button>
-              </Tooltip>
-            )}
-            {index !== 0 && (
-              <Tooltip title={t('common.move_up')} sx={{ gridArea: 'up' }}>
-                <Button onClick={() => moveItem(index - 1)}>
-                  <ArrowUpwardIcon />
-                </Button>
-              </Tooltip>
-            )}
-          </Box>
-        )}
+        <Box sx={{ display: 'grid', gridTemplateAreas: '"down up"', gridTemplateColumns: '1fr 1fr', maxWidth: '8rem' }}>
+          <Tooltip title={t('common.move_down')} sx={{ gridArea: 'down' }}>
+            <Button disabled={index === 0 || index === maxIndex} onClick={() => moveItem(index + 1)}>
```

Comment:
```suggestion
            <Button disabled={index === maxIndex} onClick={() => moveItem(index + 1)}>
```
må kunne flytte den første nedover også 
![bilde](https://user-images.githubusercontent.com/6313468/225233412-951e230d-a8b5-4b05-8146-7ac9c6be3acf.png)